### ISSUE TITLE
testicular artery morphology

### ISSUE THREAD

**Issue by slaulederkind** [obophenotype/mammalian-phenotype-ontology#3193]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1.  abnormal testicular artery morphology
2.  increased wall to lumen ratio of testicular artery
3.  decreased wall to lumen ratio of testicular artery

>2. Synonyms (e.g., absent spleen)

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1.  any structural anomaly of the paired branches of the abdominal aorta that supplies blood to the testes.
2.  increased wall-to-lumen ratio is the ratio of vascular wall thickness to luminal diameter, which is one of the important parameters in the field of vascular medicine because it indicates the character of vascular wall
3.  decreased wall-to-lumen ratio is the ratio of vascular wall thickness to luminal diameter, which is one of the important parameters in the field of vascular medicine because it indicates the character of vascular wall

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID:11693268

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1.  abnormal systemic artery morphology (MP:0011655)
2.  abnormal testicular artery morphology (new)
3.  abnormal testicular artery morphology (new)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

RGD:gthayman

**Comment by anna-anagnostop** [comment]
Added new terms:
abnormal testicular artery morphology MP:0031194
decreased wall-to-lumen ratio of testicular artery MP:0031195
increased wall-to-lumen ratio of testicular artery MP:0031196

**Comment by anna-anagnostop** [comment]
Re-opened issue to change primary term labels for MP:0031195 and MP:0031196 to "decreased testicular artery wall-to-lumen ratio" and "increased testicular artery wall-to-lumen ratio", respectively.
